bundles/org.openhab.binding.draytonwiser/README.md

```diff
@@ -0,0 +1,248 @@
+# Drayton Wiser Binding
+
+This binding integrates the [Drayton Wiser Smart Heating System](https://wiser.draytoncontrols.co.uk/).
+The integration happens through the HeatHub, which acts as an IP gateway to the ZigBee devices (thermostats and TRVs).
+
+## Supported Things
+
+The Drayton Wiser binding supports the following things:
+
+* Bridge - The network device in the controller that allows us to interact with the other devices in the system
+* Boiler Controller - The _HeatHub_ attached to the boiler. This also acts as the hub device.
+* Rooms - Virtual groups of _Room Thermostats_ and _TRVs_ that can have temperatures and humidities
+* Room Thermostats - Wireless thermostats which monitor temperature and humidity, and call for heat
+* Smart TRVs - Wireless TRVs that monitor temperature, alter the radiator valve state and call for heat
+* Hot Water - Virtual thing to manage hot water states
+* Smart Plugs - Wireless plug sockets which can be remotely switched
+
+## Discovery
+
+The HeatHub can be discovered automatically via mDNS, however the `secret` cannot be determined automatically.
+Once the `secret` has been configured, all other devices can be discovered by triggering device discovery again.
+
+## Binding Configuration
+
+None required
+
+## Thing Configuration
+
+### HeatHub Configuration
+
+Once discovered, the HeatHub `secret` needs to be configured.
+There are a few ways to obtain this, assuming you have already configured the system using the Wiser App.
+
+1. Temporarily install a packet sniffing tool on your mobile device. Every request made includes the `secret` in the header.
+2. Enable setup mode on the HeatHub. Connect a machine temporarily to the `WiserHeat_XXXXX` network and browse to `http://192.168.8.1/secret` to obtain the `secret`.
+
+The `refresh` interval defines in seconds, how often the binding will poll the controller for updates.
+
+The `awaySetPoint` defines the temperature in degrees Celsius that will be sent to the heathub when away mode is activated.
+
+## Channels
+
+### Readonly Channels
+
+#### Boiler Controller
+
+| Channel                      | Item Type            | Description                                              |
+|------------------------------|----------------------|----------------------------------------------------------|
+| `heatingOverride`            | Switch               | State of the heating override button on the controller   |
+| `heatChannel1Demand`         | Number:Dimensionless | Current demand level of heating channel 1                |
```

Comment:
Is this percent?

Reply:
Yes this is generaly used for readonly percentage channels.